### PR TITLE
fix(ui): Allow a name to specified when creating an EC2 cluster

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: src/workflows/pyfr/common/steps/Visualization/Start/index.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ const VisualizationStart = React.createClass({
       payload.cluster = ClusterPayloads.tradClusterPayload(this.state[this.state.serverType].profile);
     } else if (this.state.serverType === 'EC2') {
       payload.cluster = ClusterPayloads.ec2ClusterPayload(
-        this.state[this.state.serverType].profile.name,
+        this.state[this.state.serverType].name,
         this.state[this.state.serverType].machine,
         this.state[this.state.serverType].clusterSize,
         this.state[this.state.serverType]._id
```
Comment (cjh1) [inline src/workflows/pyfr/common/steps/Visualization/Start/index.js]: @TristanWright Shouldn't this be:

```javascript
this.state[this.state.serverType].profile._id
```

**Comment by cjh1** [comment]
@TristanWright  PTAL

**Comment by TristanWright** [comment]
trying out..

**Comment by TristanWright** [comment]
hmmm yeah the ec2 profile object is inconsistent, sometimes its `profile._id` sometimes it's just `.profile`

**Comment by TristanWright** [comment]
I tried each one seems to be `.profile`  each time. 

- Improved errors in the payload util
- missing rerun button on pv/view.

**Pull request merged**